### ISSUE TITLE
Punctuation between literary source references

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/MWS#54]
There is a persistent bug in the digitization of mw that was first mentioned [here](https://github.com/sanskrit-lexicon/MWS/issues/45#issuecomment-341602833) and discussed further in that issue #45.

In the current digitization, essentially all punctuation ( usu. comma or semicolon) is missing as separators between literary source references.

It is certainly possible to correct this situation, but because of the large number of such literary source references in this dictionary, a complete correction would require many hours of human attention.
In other words, correction is a hard problem.

This issue will identify this as an unsolved problem, and will provide a place for detailing work done to solving or mitigating the problem.

**Comment by funderburkjim** [comment]
### a semicolon bandaid
As also mentioned in #45, the display for mw has be modified to insert a semicolon between consecutive literary source references.   This is not a change to the digitization mw.xml, but just to the display.

This change could be done at the level of mw.xml.

While this change often results in retrieving the punctuation of the printed text,  it also often still does not get the right punctuation, since the right punctuation is often a comma.

**Comment by Andhrabharati** [comment]
This will be taken care of in my current full review, and the issue does not need any more discussion.
